test/sqllogictest/aclitem.slt

```diff
@@ -309,7 +309,7 @@ SELECT mz_internal.mz_aclexplode(ARRAY[mz_internal.make_mz_aclitem('u1', 'u2', '
 (u2,u1,SELECT,f)
 
 query error MZ_ACL arrays must not contain null values
-SELECT mz_internal.mz_aclexplode(array[null]::mz_aclitem[]);
+SELECT mz_internal.mz_aclexplode(array[null]::mz_catalog.mz_aclitem[]);
```

Comment:
These didn't _have_ to change, right? They're still visible via the default search path?

Reply:
No they didn't have to, they are still visible via the default search path.

Reply:
Great, just wanted to make sure I was understanding the change correctly.

Reply:
Quick sanity check:

```
materialize=> SELECT mz_internal.mz_aclexplode(array[null]::mz_aclitem[]);
ERROR:  MZ_ACL arrays must not contain null values
materialize=> SELECT mz_internal.mz_aclexplode(array[]::mz_aclitem[]);
 mz_aclexplode 
---------------
(0 rows)

```